File: app/src/main/java/com/neome/api/meta/base/dto/DtoLogTree.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DtoLogTree extends DtoLogItem
{
  public String bgColor;

  @Nullable
  public DtoLogTreeItem[] children;

  public String keyColor;

  public long keyWidth;

  public String lineColor;

  public long tabWidth;

  public String valueColor;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DtoLogTree.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DtoLogItem
import com.neome.api.meta.base.dto.DtoLogTreeItem

interface DtoLogTree : DtoLogItem
{
  val bgColor: String
  val children: List<DtoLogTreeItem>?
  val keyColor: String
  val keyWidth: Long
  val lineColor: String
  val tabWidth: Long
  val valueColor: String
}